src/main/java/com/example/controller/AuthController.java

```diff
@@ -19,11 +22,18 @@
 @RequiredArgsConstructor
 public class AuthController {
     private final UserService userService;
+    private final AuthService authService;
 
     @Operation(summary = "User registration", description = "New user registration")
     @PostMapping("/registration")
     public UserResponseDto register(@RequestBody @Valid UserRegistrationRequestDto requestDto)
             throws RegistrationException {
         return userService.register(requestDto);
     }
+
+    @Operation(summary = "User login", description = "New user registration")
+    @PostMapping("/login")
+    public UserLoginResponseDto login(@RequestBody @Valid UserLoginRequestDto requestDto) {
```

Comment:
Why these changes in Category PR ?

Reply:
I don't remember. Maybe I added @Valid annotation